src/entrypoints/dead-clicks-autocapture.ts

```diff
@@ -205,7 +209,9 @@ class LazyLoadedDeadClicksAutocapture implements LazyLoadedDeadClicksAutocapture
                 this._config.selection_change_threshold_ms
             )
             const mutationTimeout = checkTimeout(click.mutationDelayMs, this._config.mutation_threshold_ms)
-            const absoluteTimeout = checkTimeout(click.absoluteDelayMs, this._config.mutation_threshold_ms)
+            // we want to timeout eventually even if nothing else catches it...
+            // we leave a little longer than the maximum threshold to give the other checks a chance to catch it
+            const absoluteTimeout = checkTimeout(click.absoluteDelayMs, this._config.mutation_threshold_ms * 1.1)
```

Comment:
we saw absolute timeouts, without seeing mutation timeouts, that also seems wrong so...